printer_mods/README.md

```diff
@@ -339,4 +339,4 @@ like so:
 | | [V0 Skirt set](./Zen3D/V0_Skirt)| SSR, PowerSwitch, Nameplate and color change printing  | :heavy_check_mark: |:x: |:x: |:x:|
 | | [V0 1515 T-nut](./Zen3D/V0_1515_T-nut)| M3 square and hex nuts for Voron0 1515 frame | :heavy_check_mark: |:x: |:x: |:x:|
 | zruncho | [V0 No-Drop Nuts](./zruncho/V0_No_Drop_Nuts) | Voron0 nut inserts which auto-align (LDO frames only) | :heavy_check_mark: | :x: | :x: | :x: |
-
+| Edwardlining | [V2.4 exhaust](./Edwardlining/Voron2.4_exhaust_filter_Duct) | Voron2.4 exhaust filter Duct | :x: |:grey_question: |:heavy_check_mark:| :x: | :x:| 
```

Comment:
Mods should be listed in alphabetical order (by username). This should come after `eddie`